src/components/NewWorkspaceModal.test.js

```diff
@@ -173,4 +199,128 @@ describe('NewWorkspaceModal', () => {
       )
     ).toBeNull();
   });
+
+  it('shows an option for Workspace will have protected data (enhanced bucket logging) if a Google billing project is selected', async () => {
+    // Arrange
+    const user = userEvent.setup();
+
+    Ajax.mockImplementation(() => ({
+      Billing: {
+        listProjects: async () => [gcpBillingProject, azureBillingProject],
+      },
+      ...nonBillingAjax,
+    }));
+
+    await act(async () => {
+      // eslint-disable-line require-await
+      render(
+        h(NewWorkspaceModal, {
+          cloneWorkspace: false,
+          onSuccess: () => null,
+          onDismiss: () => null,
+          customMessage: null,
+          requiredAuthDomain: false,
+          title: null,
+          buttonText: null,
+          // workflowImport: false <== Not specified. False should be the default
+        })
+      );
+    });
+
+    const projectSelector = screen.getByText('Select a billing project');
+    await user.click(projectSelector);
+
+    const googleBillingProject = screen.getByText('Google Billing Project');
+    await user.click(googleBillingProject);
+
+    // Assert
+    // getByText throws an error if the element is not found:
```

Comment:
In this test, it would be good to assert that the checkbox is not disabled.

(I can also imagine filling out all the fields, selecting the checkbox, clicking on "Create Workspace", and spying on what is sent for the Ajax request to ensure that the protected data option is sent, but I don't see that any of the existing tests go to that level of testing, and `NewWorkspaceModal` may not be in a state that would make such a test straightforward.)

Reply:
Added an assertion here as suggested. 

Yeah, while I agree it would be good to test that the option is actually being sent, I am not sure this is the moment to add that test. I would prefer that to be a test for the Ajax request function itself so I'd be reluctant to add it here.